upload_to_sheets.py

```
@@ -378,11 +497,99 @@ def process_consolidated_data(consolidated_file):
+
+# Add successful local builds section if we have any
+if has_successful_local_builds:
+    logging.info("Attempting to add Local Build Success section to summary data")
+    successful_builds_sheet_id = get_worksheet_id_with_retry(spreadsheet, "Local Build Success")
+
+    if successful_builds_sheet_id:
+        logging.info("Adding Local Build Success section to summary data")
+        summary_data.extend([
+            ["", "", "", "", "", ""],
+            ["Local Build Success", "", "", "", "", ""],
+            ["PRs that build locally but fail on Jenkins", successful_builds_count, "", "", "", ""],
```

Ruff (0.8.2)

Undefined name `successful_builds_count`

(F821)